build.gradle

```diff
@@ -539,30 +560,123 @@ publishing {
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'cschaebe'
+                        name = 'Christoph Schaebel'
+                        email = 'christoph.schaebel@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
                     }
                     developer {
                         id = 'lbrandl'
                         name = 'Lukas Brandl'
-                        email = 'lukas.brandl@dc-square.de'
-                        organization = 'dc-square GmbH'
-                        organizationUrl = 'https://www.dc-square.de'
+                        email = 'lukas.brandl@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
                     }
                     developer {
                         id = 'flimpoeck'
                         name = 'Florian Limpoeck'
                         email = 'florian.limpoeck@dc-square.de'
-                        organization = 'dc-square GmbH'
-                        organizationUrl = 'https://www.dc-square.de'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+                    developer {
+                        id = 'sauroter'
+                        name = 'Georg Held'
+                        email = 'georg.held@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
+                    }
+
+                    developer {
+                        id = 'SgtSilvio'
+                        name = 'Silvio Giebl'
+                        email = 'silvio.giebl@hivemq.com'
+                        organization = 'HiveMQ GmbH'
+                        organizationUrl = 'https://www.hivemq.com'
                     }
                 }
                 scm {
                     connection = 'scm:git:git@github.com:hivemq/hivemq-community-edition.git'
                     developerConnection = 'scm:git:git@github.com:hivemq/hivemq-community-edition.git'
                     url = 'https://github.com/hivemq/hivemq-community-edition/tree/master'
                 }
+
+            }
+
+            pom.withXml {
+                asNode().dependencies.dependency.each { dep ->
+                    if (dep.artifactId.last().value().last() in ["logback-classic", "slf4j-log4j12"]) {
+                        assert dep.parent().remove(dep)
+                    }
+                }
+            }
+        }
+
+        sources(MavenPublication) {
+            from components.java
+            artifactId 'hivemq-community-edition-embedded'
+
+            artifact sourceJar {
+                classifier "sources"
             }
         }
```

Comment:
This makes no sense, the sources jar is part of the publication and has not its own publication

Reply:
👍